src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -156,17 +156,16 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
   def getWorkspace(workspaceName: WorkspaceName): Future[PerRequestMessage] =
     dataSource.inFutureTransaction(readLocks=Set(workspaceName)) { txn =>
       withWorkspaceContext(workspaceName, txn) { workspaceContext =>
-        gcsDAO.getMaximumAccessLevel(userInfo.userEmail,workspaceContext.workspace.workspaceId) flatMap { accessLevel =>
-          if (accessLevel < WorkspaceAccessLevels.Read)
-            Future.successful(RequestComplete(ErrorReport(StatusCodes.NotFound, noSuchWorkspaceMessage(workspaceName))))
-          else {
-            gcsDAO.getOwners(workspaceContext.workspace.workspaceId) map { owners =>
-              val response = WorkspaceListResponse(accessLevel,
-                workspaceContext.workspace,
-                getWorkspaceSubmissionStats(workspaceContext, txn),
-                owners)
-              RequestComplete(StatusCodes.OK, response)
-            }
+        val accessLevel = containerDAO.authDAO.getMaximumAccessLevel(RawlsUserRef(RawlsUserSubjectId(userInfo.userSubjectId)), workspaceContext.workspace.workspaceId, txn)
```

Comment:
For now I think we should do what works. I think once the dust settles on this feature branch we should go through and evaluate Future-y stuff and refactor the google services dao.